content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Elastic Contributes its Continuous Profiling Agent to OpenTelemetry
+linkTitle: Elastic Contributes Profiling Agent # Mandatory, make sure that your short title.
+date: 2024-06-06 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Bahubali Shetti](https://github.com/bshetti) (Elastic), [Alexander
+  Wert](https://github.com/AlexanderWert) (Elastic), [Morgan
+  McLean](https://github.com/mtwo) (Splunk), [Ryan
+  Perry](https://github.com/Rperry2174) (Grafana)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+issue: https://github.com/open-telemetry/community/issues/1918
+sig: Profiling SIG
+---
+
+Following significant collaboration between
+[Elastic](https://www.elastic.co/observability-labs/blog/elastic-donation-proposal-to-contribute-profiling-agent-to-opentelemetry)
+and [OpenTelemetry's profiling community](./profiling.md), which included a
```

Comment:
```suggestion
and [OpenTelemetry's profiling community](/blog/2024/profiling/), which included a
```

I asssume you want to link to the profiling blog post?

Reply:
yes